modules/ROOT/pages/file/file-read.adoc

```diff
@@ -6,10 +6,9 @@ endif::[]
 
 Using Anypoint Connector for File (File Connector), the transport can read a file at any point in the flow, rather than only as a result of inbound endpoint polling in a Mule 3 implementation. 
 
-The only required parameter for the `Read` operation is `path`. This can be either the full path to the
-file or a relative path to the working directory value defined in the configuration
-referenced by the `config-ref` parameter. For an example of setting a configuration, see
-xref:file/file-connector.adoc#connection_settings[Connection Settings]. 
+The only required parameter for the `Read` operation is the path to the file to be read. This can be either 
+the full path to the file or a relative path to the working directory value defined in the configuration
```

Comment:
I don't think you need the word "value." The sentence is long, so it would be good to shorten it.